fsr.ino

```diff
@@ -611,6 +616,93 @@ class SerialProcessor {
     Serial.print("\n");
   }
 
+  const int16_t SAVE_SLOT_TAKEN_MARKER = -42;  // any negative number would do, as actual thresholds are >=0
```

Comment:
I think all this eeprom code should live in its own class (say `EepromProcessor`), and have an instance of it within the SerialProcessor. Seems like only the PersistThresholds and RestoreThresholds need to be public functions and the rest can be private.

Also slight lean towards `SaveThresholds` over `PersistThresholds`. Save sounds a bit more intuitive to me.